src/styles.scss

```diff
@@ -46,14 +46,15 @@ body {
 
   // STYLING FOR TABLE LIST ENTRIES
   .list-item {
-    display: grid;
+    display: flex;

```

Comment:
I think it will be easier to implement this design by using CSS grid rather than flexbox.